dice-engine/README.md

```diff
@@ -33,17 +33,25 @@ problem in Clojure.
 
 **Dice-roll notation:** `NdX`
 
+**Die map representation**
+
+```clojure
+(def die {:value           1
```

Comment:
Is this a dice or is this an instance of a roll?

Reply:
It's an instance of a single die